blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -41,79 +50,176 @@ industry. We are committed to making a positive impact on the community
 through our work, and we look forward to continuing our open-source
 contributions in the future.
 
-## Our contributions:
+## Our contributions
 
-### coreboot
+![coreboot logo](/covers/coreboot-logo.svg)
 
-coreboot is an open-source firmware that provides a lightweight, secure,
+[coreboot](https://3mdeb.com/training/#coreboot-for-embedded-linux-developers)
+is an open-source firmware that provides a lightweight, secure,
 and fast boot experience for PCs, laptops, servers, and embedded devices.
+The changes made by our engineers can be divided into three main groups.
+
+#### Adding support for new platforms and maintaining existing ones
+
+Recently, our engineers have focused on expanding coreboot support for the
+latest platforms, it is worth mentioning that Karol Zmyslowski has added
+[support for Jasper Lake](https://review.coreboot.org/c/coreboot/+/73934).
+
+Michał Kopeć and Michał Żygowski had a major impact on Alder Lake support in
+coreboot. Michał Kopeć has added [support for Alder Lake](https://review.coreboot.org/c/coreboot/+/63374)
+and Michał Żygowski in the [set of patches](https://review.coreboot.org/c/coreboot/+/68449/11)
+has added improvements huge improvements. Also, Michał Żygowski made a significant
+contribution to [Tiger Lake support](https://review.coreboot.org/c/coreboot/+/56171).
+
+In the previous year, our team also added
+[support for MSI](https://review.coreboot.org/c/coreboot/+/68078) PRO Z690-A
+WIFI and MSI PRO Z690-A in DDR4 and DDR5 versions.
+
+Krystian Hebel, Sergii Dmytruk, and Yaroslav Kurlaev have made major contributions
+to developing and maintaining support for Power platforms. Their changes include
+[emulation improvements](https://review.coreboot.org/c/coreboot/+/67061),
+and fixing ongoing issues related to these platforms.
+
+#### Platform security (TPM, TXT)
+
+Our team also worked on [TPM](https://3mdeb.com/shop/modules/tpm-2-0/) and
+[TXT](https://blog.3mdeb.com/2022/2022-03-17-optiplex-txt/) solutions that
+could support the security of coreboot-based platforms. Sergii Dmytruk added
+quite a few improvements related to
+[TPM support](https://review.coreboot.org/c/coreboot/+/68748),
+while Michal Zygowski added some
+[TXT improvements](https://review.coreboot.org/c/coreboot/+/59519).
```

Comment:
This needs a more in-depth explanation of why is the value of those changes.

Reply:
Deleted this section and now we're linking to the 4.20 release blog. https://github.com/3mdeb/news-and-ideas/commit/68a72cdebcee326365faf61a3b7861f718765122#diff-5dff579917e1df023c3d68f093f637f7fb497ee2f22650cc786ac30a195a7eceR79